src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer.Xaml/Microsoft.CodeAnalysis.LanguageServer.Xaml.csproj

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.CodeAnalysis.LanguageServer.Xaml</RootNamespace>
```

Comment:
> <RootNamespace>Microsoft.CodeAnalysis.LanguageServer.Xaml</RootNamespace> [](http://example.com/codeflow?start=3&length=74)

Should this project be gone now?

Reply:
Yes, thanks. Apparently deleting a project in VS doesn't mean what I thought it meant. :-)